lib/active_merchant/billing/gateways/securion_pay.rb

```diff
@@ -79,6 +79,29 @@ def verify(credit_card, options={})
         end
       end
 
+      def store(credit_card, options = {})
+        if options[:customer_id].blank?
+          MultiResponse.run() do |r|
+            #create charge object
+            r.process { authorize(100, credit_card, options) }
+            #create customer and save card
+            r.process { create_customer_add_card(r.authorization, options) }
+            #void the charge
+            r.process(:ignore_result) { void(r.params["metadata"]["chargeId"], options) }
+          end
+        else
+          verify(credit_card, options)
```

Comment:
I'm curious how doing a `verify` will automatically store the card?

What does the active merchant response look like from verify?